src/data/datamapping.cpp

```diff
@@ -150,13 +209,14 @@ std::string generateHashString(const DataMapping& dm) {
     unsigned int excludeColumnsHash = ghoul::hashCRC32(a);
 
     return fmt::format(
-        "DM|x{}|y{}|z{}|name{}|m{}|{}|{}",
-        dm.xColumnName.value_or(""),
-        dm.yColumnName.value_or(""),
-        dm.zColumnName.value_or(""),
-        dm.nameColumn.value_or(""),
+        "DM|{}|{}|{}|{}|{}|m{}|{}|{}",
```

Comment:
Could use some tips on how to format this.  I end up changing my mind.  

But this way, with the hashing for each column name, seems more robust than just using the names as is